jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -73,6 +73,10 @@ import java.io.File
 import java.time.Instant
 import java.util.concurrent.atomic.AtomicBoolean
 import javax.swing.Icon
+import org.jsoup.Jsoup;
```

Comment:
we don't need yet another deserialization library

take a look at JDOM in `com.intellij.configurationStore` (preferred)
or pull in jackson-xml instead